src/Table.js

```diff
@@ -519,42 +524,49 @@ export class Table {
 
     async execute(model, op, cmd, properties = {}, params = {}) {
         let mark = new Date()
-        let trace = {model, cmd, op, properties}
+        let trace = { model, cmd, op, properties }
         let result
         try {
             let client = params.client || this.client
             if (params.stats || this.metrics || this.monitor) {
                 cmd.ReturnConsumedCapacity = params.capacity || 'INDEXES'
                 cmd.ReturnItemCollectionMetrics = 'SIZE'
             }
-            this.log[params.log ? 'info' : 'trace'](`OneTable "${op}" "${model}"`, {trace})
-            if (this.V3) {
-                result = await client[op](cmd)
+            if (op === 'get') {
+                result = await this.loader.load(cmd)
             } else {
```

Comment:
here is the main change, instead of directly executing the get operation it delegates the get to the loader that will accumulate the gets and perform a single batch get.